cli/build.go

```diff
@@ -431,12 +438,6 @@ func BuildInsideContainer(target BuildTarget, containerOpts BuildContainerOpts,
 		} else {
 			localYbPath = p
 		}
-		// After downloading YB and uploading it to the container, we should delete it locally
```

Comment:
Without removing `.yb-tmp/` we'll have to add this to every `.gitignore` that uses the CLI... Why did you remove this?

Reply:
Changed, see below / commit comment